lib/mix/tasks/phoenix.gen.model.ex

```diff
@@ -183,8 +204,9 @@ defmodule Mix.Tasks.Phoenix.Gen.Model do
 
   defp types(attrs) do
     Enum.into attrs, %{}, fn
-      {k, {c, v}} -> {k, {c, value_to_type(v)}}
-      {k, v}      -> {k, value_to_type(v)}
+      {k, {:unique, v}} -> {k, value_to_type(v)}
```

Comment:
Do we need this? We extract the uniques out of `attrs` before we call `types/1`.  :)